cpu/stm32f4/include/periph_cpu.h

```diff
@@ -112,7 +112,7 @@ void gpio_init_af(gpio_t pin, gpio_af_t af);
  */
 static inline void dma_poweron(int stream)
 {
-    if (stream < 8) {
+    if (stream <= 8) {
```

Comment:
Wait... but then the if statement at line 187 is wrong as well... that's how I made the mistake...

https://github.com/RIOT-OS/RIOT/pull/4971/files#diff-2e30e656008f8f19435be34c94f43addR187